src/components/Publish/Navigation/index.module.css

```diff
@@ -1,10 +1,12 @@
 .navigation {
   margin: -1.3rem calc(var(--spacer) / -1.5) var(--spacer)
     calc(var(--spacer) / -1.5);
-  padding: calc(var(--spacer) / 2) 0;
   scrollbar-width: none;
   border-bottom: 1px solid var(--border-color);
   background-color: var(--background-highlight);
+  width: calc(100% + var(--spacer) 0 * 2);
```

Comment:
issue(blocking): Is this needed for the functionality? width > 100% does not look correct to me. Also the math at the end looks weird (`0 * 2`).

suggestion: remove width here or simply keep at 100%